tests/e2e/run.sh

```diff
@@ -31,7 +31,6 @@ test_e2e() {
     setup_deployment_env false false
     remove_existing_stackrox_resources
     setup_default_TLS_certs
-    "$ROOT/tests/complianceoperator/create.sh"
```

Comment:
The Compliance Operator installation is handled on line 36 instead.

Reply:
I'm pretty sure this was installed this way to allow the compliance operator go tests to run under GKE. And I think there are other PRs to address running this under OCP, it will need to be excluded from the nongroovy tests job i.e. 
[gke-nongroovy-e2e-tests](https://prow.ci.openshift.org/job-history/gs/origin-ci-test/pr-logs/directory/pull-ci-stackrox-stackrox-master-gke-nongroovy-e2e-tests)

Reply:
Ack - I took another approach that attempts to preserve both paths for the time being.

Reply:
It took me a while to unwind why this existed. At first, I thought it was around by mistake and what we really wanted was to test the CRs as they are generated by the compliance operator. However, even though these CRDs are technically owned by the Compliance Operator, ACS tests are installing them and generating some mock data to test against.

I documented this relationship in tests/complianceoperator/README.md, and left it as is so that the existing v1 compliance testing will continue to work as it.

Then I pivoted this patch to focus on setting up a separate test runner for install CO on a secure cluster (and not the mocked CO data), so that we can test the full integration between CO and ACS in a separate test suite.